src/platform/testing/visual-regression/index.js

```diff
@@ -109,9 +109,9 @@ async function setup() {
   try {
     await beginApplication(browser);
   } catch (e) {
-  /* eslint-disable */
+    /* eslint-disable */
```

Comment:
I was figuring out how this worked and my editor added these indents. :man_shrugging: